src/templates/verilogeditor.js

```diff
@@ -55,7 +55,7 @@ class Verilog_editor {
     str += this.set_Constants(space, structure['generics']);
     str += '\n';
     str += '  // Ports\n';
-    str += this.set_Signals(space, structure['ports']);
+    str += this.set_Signals_Tb(space, structure['ports']);
```

Comment:
Try to use lower case.

Reply:
I have no problems using lower case. But as you know we've changed the style. So it means changing the whole file.
No problem at all but just to let you know.

Reply:
Yes, but in the new code added we should use the new style.

Reply:
I agree, but this is old code and bug fixing. Not a refactoring.